packages/itmat-job-executor/test/unit_tests/csvcurator.test.ts

```diff
@@ -194,18 +196,19 @@ describe('CSVCuratorClass', () => {
     it('csvcurator uploads csv file okay', async () => {
         const readStream = fs.createReadStream('./test/testFiles/CSVCurator.tsv');
         const mongoStub = new MongoStub();
+        const jobEntry = stub<IJobEntry<any>>({  // subset of the IJobEntry interface
```

Comment:
try to replace IJobEntry<any> with IJobEntryForDataCuration

Reply:
similarly there is another interface called IJobEntryForFieldCuration in the same file. you can look in the shape for the object to determine whether to replace IJobEntry<any> with IJobEntryForDataCuration or IJobEntryForFieldCuration